pkg/skaffold/kubernetes/portforward/forwarder_manager.go

```diff
@@ -107,11 +109,15 @@ func (p *ForwarderManager) Start(ctx context.Context, namespaces []string) error
 		return nil
 	}
 
+	eventV2.TaskInProgress(constants.PortForward, eventV2.GetIteration())
```

Comment:
This kind of looks a but odd. Shd we just change all the other uses of `eventV2.TaskInProgress` to use `eventV2.GetIteration()`?
```suggestion
	eventV2.TaskInProgress(constants.PortForward)
```

Reply:
Good point, or I can just change all `Task*()` functions to use the underlying handler's iteration value and remove the extra parameter altogether. The handler's iteration field could then just increment when it sees `constants.DevLoop`

Reply:
yeah. Great idea. We don't even need the `devIteration` in runner.